ds-caselaw-ingester/tests.py

```diff
@@ -41,6 +66,34 @@ def test_handler_messages(
 
         log = capsys.readouterr().out
         assert "Ingester Start: Consignment reference DXW-2001-DRGN" in log
+        assert "v1: True" in log
+        assert "Ingesting document" in log
+        assert "Updated judgment xml" in log
+        assert "Upload Successful" in log
+        assert "Ingestion complete" in log
+
```

Comment:
Not for this PR necessarily, but it would be good to have a test that tests the expected functionality of the lambda on top of the log messages.

Could use `moto` to mock the aws session to show that given some mock s3 bucket we read from, we save to another s3 bucket, send out the sns notifications etc.

Reply:
:+1: